### PR TITLE
Take over PR640 Adding command to copy meta files into dist dir

### LINKED ISSUE #635: README and LICENSE are missing from the npm package
Until [0.8.11](https://www.npmjs.com/package/solc/v/0.8.11) the package page on npm had our README. In [0.8.12](https://www.npmjs.com/package/solc/v/0.8.12) README is no longer there. LICENSE file is not there either.

I think they disappeared because we now package only files from the `dist/` subdir. Any extra files need to be explicitly copied there.

We need go add these two back. Do we need any other files besides them?

### PR DESCRIPTION
**Pull request by matheusaaguiar**
Closes #640 and Fixes #635 .

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "build": "tsc",
-    "postbuild": "node build/postbuild.js",
+    "postbuild": "node build/postbuild.js && cp README.md ./dist/README.md && cp LICENSE ./dist/LICENSE",
```
Comment (cameel) [inline package.json:11]:
```suggestion
    "postbuild": "node build/postbuild.js && cp README.md LICENSE dist/",
```

**Comment by cameel** [comment]
What about https://github.com/ethereum/solc-js/pull/640#discussion_r921436053? Have you checked what other files we might need?

Also, since this PR replaces the other one, you can add `Closes #640` in the description. This way we get a link between them and also github will close that other PR when we merge this one. Also `Fixes #635` since it fixes the issue (it should have been in the original PR too).

**Comment by cameel** [comment]
As for the CI failure, I need to go back to #633. Maybe you'd like to try out [@ekpyron's suggestion](https://github.com/ethereum/solc-js/pull/633#pullrequestreview-1058447617) and submit PR?

File: package.json (hunk 1 above)
Comment (matheusaaguiar) [reply package.json]: Done.

**Comment by matheusaaguiar** [comment]
> As for the CI failure, I need to go back to #633. Maybe you'd like to try out [@ekpyron's suggestion](https://github.com/ethereum/solc-js/pull/633#pullrequestreview-1058447617) and submit PR?

It seems that the current version of Debian in CI (buster 10.12) does not have the package `python-is-python3`.

**Comment by cameel** [comment]
> It seems that the current version of Debian in CI (buster 10.12) does not have the package `python-is-python3`.

ok. Thankfully looks like the new image I'm switching to in #633 is actually based on Ubuntu and does have `python-is-python3`. I'm going to include the fix there.

**Comment by cameel** [comment]
Also, I wanted to rebase this PR but looks like now I have permission problems :) Apparently you have not checked the checkbox that allows maintainers to push to the PR branch. I can push to the original PR so I'm going to merge it instead and close this one.

**Pull request closed without merging**